lib/rdoc/markup/to_html.rb

```diff
@@ -98,7 +98,12 @@ def handle_RDOCLINK url # :nodoc:
 
       gen_url CGI.escapeHTML(url), CGI.escapeHTML(text)
     when /^rdoc-image:/
-      %[<img src=\"#{CGI.escapeHTML($')}\">]
+      url, alt = $'.split(":", 2) # Split the string after "rdoc-image:" into url and alt
+      if alt
```

Comment:
I think we can omit creating `alt=""` if alt is an empty string

Reply:
Makes sense.